### PR TITLE
Enable underlining hyperlink on hover even if not focused

### LINKED ISSUE #8475: Underline hyperlink on hover, even if not focused
# Description of the new feature/enhancement

I'd like Windows Terminal to underline the hyperlink on which I hover the mouse pointer, even if the pane and the Windows Terminal window do not have the focus. This would feel more consistent because a single Ctrl+click already suffices to open the hyperlink regardless of where the focus is.

I suppose it should then show the tool tip as well.

### PR DESCRIPTION
**Pull request by Hegunumo**
Closes #8475

### CODE REVIEW

**Review by amanda008868 — APPROVED** [review]

**Comment by DHowett** [comment]
What happened? We aren’t ignoring you, the team is on vacation for the rest of the year. You can leave your pull request open and we will review it when we are back in the office!

File: src/cascadia/TerminalControl/TermControl.cpp [hunk 1]
```diff
@@ -1344,8 +1344,10 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
 
         const auto ptr = args.Pointer();
         const auto point = args.GetCurrentPoint(*this);
+        const auto cursorPosition = point.Position();
+        const auto terminalPosition = _GetTerminalPosition(cursorPosition);
```
Comment (PankajBhojwani) [inline src/cascadia/TerminalControl/TermControl.cpp:1348]: nit: with these 2 declarations here, the `cursorPosition` and `terminalPos` declarations below (lines 1369 and 1412) can be removed

**Review by zadjii-msft — APPROVED** [review]
Looks fine to me, but I agree with Pankaj. We should remove the redundant declarations below now.

**Comment by DHowett** [comment]
I made the requested changes. This PR is 20 days old and I don't want to make @Hegunumo spend another cycle on it. :smile:

**Review by PankajBhojwani — APPROVED** [review]

**Pull request merged**